web/profiles/joinup/joinup.profile

```diff
@@ -220,6 +220,10 @@ function joinup_form_node_form_alter(&$form, FormStateInterface $form_state, $fo
   $form['revision']['#access'] = FALSE;
   $form['meta']['#access'] = FALSE;
 
+  /** @var \Drupal\Core\Entity\EntityFormInterface $form_object */
+  $form_object = $form_state->getFormObject();
+  $form['uid']['#access'] = \Drupal::currentUser()->hasPermission('administer nodes') && ($form_object->getEntity()->bundle() === 'tallinn_report');
```

Comment:
This can be easily transferred to the hook in the tallinn module that you already created above. This will also allow us to maintain the specific community functionality all tied with the tallinn module.